test/apps/app/public/static/index.html

```diff
@@ -109,7 +109,7 @@
   
       function logout(e) {
         e.preventDefault();
-        authClient.signOut()
+        authClient.signOut({ clearTokensBeforeRedirect: true })
```

Comment:
This is because we are not running `start()` by default, right? Can you add a small comment explaining why we are passing true here.